File: QTD2.Domain/Entities/Core/ILA_Topic_Link.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using QTD2.Domain.Entities.Common;

namespace QTD2.Domain.Entities.Core
{
    public class ILA_Topic_Link : Entity
    {
        public int ILAId { get; set; }

        public int ILATopicId { get; set; }

        public virtual ILA ILA { get; set; }

        public virtual ILA_Topic ILA_Topic { get; set; }

        public ILA_Topic_Link(ILA ila, ILA_Topic ilaTopic)
        {
            ILAId = ila.Id;
            ILATopicId = ilaTopic.Id;
            ILA = ila;
            ILA_Topic = ilaTopic;
        }

        public ILA_Topic_Link()
        {
        }
    }
}

File: QTD.SPA/src/app/_DtoModels/ILA_Topic_Link/ILA_Topic_Link.ts
import { Entity } from "@models/Entity";
import { ILA } from "@models/ILA/ILA";
import { ILA_Topic } from "@models/ILA_Topic/ILA_Topic";

export class ILA_Topic_Link extends Entity {
  ilaId: string;
  ilaTopicId: string;
  ila: ILA;
  ilA_Topic: ILA_Topic;
}
